pages/common/git-sizer.md

```diff
@@ -0,0 +1,16 @@
+# git sizer
+
+> Computes various Git repository size metrics and alerts you to any that might cause problems or inconvenience.
+> Change to the directory containing your repository before running this command.
```

Comment:
Is there a reason to write this line ? You always need to run any git commands inside a git repo. That is required for _all_ git commands. I am wondering whether we should mention this explicitly since its pretty commonly known.

Reply:
I agree that it probably should be removed. It's implied, and `git sizer` returns an error anyway to notify the user when it's not run in a Git directory. 👍